packages/api-client/src/auth/ClientPreKey.ts

```diff
@@ -20,6 +20,7 @@
 import {PreKey} from '../auth/';
 
 export interface ClientPreKey {
+  /** The client ID */
```

Comment:
Do we really need these comments? Wouldn't it be better to change the variable names to be more descriptive?

- `client` -> `clienId`

Reply:
These are not variables but return values from the server. We can't just rename them.